test/visualization/test_visualization.cpp

```diff
@@ -110,8 +110,45 @@ TEST (PCL, PCLVisualizer_camera)
 //  cerr << "reset camera viewer pose:" << endl << viewer_pose << endl;
 }
 
-
-
+////////////////////////////////////////////////////////////////////////////////
+TEST (PCL, PCLVisualizer_getPointCloudRenderingProperties)
+{
+  PCLVisualizer::Ptr visualizer (new PCLVisualizer);
+
+  std::string cloud_id = "input_cloud";
+  visualizer->addPointCloud (cloud, cloud_id);
+  ASSERT_TRUE (visualizer->setPointCloudRenderingProperties (PCL_VISUALIZER_COLOR,
+                                                             1.d, 0.d, 0.d, cloud_id));
```

Comment:
I'm so surprised these .d suffixes work. I couldn't find them on cppreference. I just hope it's supported by all the compilers.

Reply:
I'm surprised that you are surprised, I used to see them (maybe it's a Fortran deformatin...). Bah, anyway, it is said that C++ assume an expression like '1.0' as a double ([check this section](https://en.wikipedia.org/wiki/C%2B%2B11#User-defined_literals)).

So I delete this suffixes.